src/api/constants.ts

```diff
@@ -61,6 +67,27 @@ export const ACCOUNT = {
       const milliseconds = 300000;
       const future = new Date(now.setMilliseconds(milliseconds));
 
+      return future;
+    },
+  },
+  // JSON web token
+  JWT: {
+    KEY: {
+      SIGNATURE: String(process.env.JWT_SIGNING_KEY),
+      ENCODING: 'base64' as BufferEncoding,
+      STRING_ENCODING: 'ascii' as BufferEncoding,
+    },
+    TOKEN: {
+      EXPIRY: '8h',
+      ALGORITHM: 'RS256' as Algorithm,
+    },
+    SESSION_EXPIRY: () => {
+      // 8 hours from now
+      const now = new Date();
+
+      const hours = 8;
+      const future = new Date(now.getTime() + hours * 60 * 60 * 1000);
```

Comment:
Constant?

Reply:
Since this is already in constants file i've extracted the seconds into it's own variable like `hours`